src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -1148,6 +1148,13 @@ RefCountedPtr<LoadBalancingPolicy::Config> ChooseLbPolicy(
   } else {
     policy_name = grpc_channel_args_find_string(resolver_result.args,
                                                 GRPC_ARG_LB_POLICY_NAME);
+    if (!LoadBalancingPolicyRegistry::LoadBalancingPolicyExists(policy_name,
+                                                                nullptr)) {
```

Comment:
I think this also needs to verify that the policy allows an empty config, because selecting the policy via channel args does not allow specifying a config.

Reply:
Fixed in the latest commit.